docs/source/ccvm_equations_of_motion.rst

```diff
@@ -0,0 +1,44 @@
+
+CCVM Simulator - Equations of Motion
```

Comment:
Well! It seems not a better choice as a title. Any suggestion?  

The idea behind this PR is to introduce the purpose of CCVM Simulators to end users. This will be done by introducing the equations and associated of CCVM model from simple to advance architecture.

Reply:
Gotcha. I think the issue isn't the name, then, but the structuring of the pages. I'll make a hierarchy so the pages are more like this:
```
- CCVM Simulator - Equations of Motion
---- DL CCVM
---- MF CCVM
---- Langevin
```
Instead of this:
```
- CCVM Simulator - Equations of Motion
- DL CCVM
- MF CCVM
- Langevin
```
Does that make sense?

Reply:
Don't forget the "-" in "DL-CCVM" and "MF-CCVM".

Reply:
I did minimal changes to the existing titles for now (just expanding names and changing the file hierarchy, see the screenshot below), but I think it would be cleaner if I removed "Stochastic Differential Equations for the" from each title, since they are all sub-pages of the "Equations of Motion" page. The downside is that when viewing a specific page, the title would only show the name of the machine. To address that, I could add a subheading to the page itself that reads "Stochastic Differential Equations", and that would keep the name in the table of contents on the left sidebar shorter.
Thoughts?
<img width="249" alt="Screen Shot 2024-02-01 at 1 13 05 PM" src="https://github.com/1QB-Information-Technologies/ccvm/assets/19154892/ced96b95-de6e-41c9-8a59-47b297c43dc5">

Reply:
This is the alternative:
<img width="1148" alt="Screen Shot 2024-02-01 at 2 50 54 PM" src="https://github.com/1QB-Information-Technologies/ccvm/assets/19154892/7a71b8f6-cdd3-4f31-afef-7f274a05223a">

I think I like this better, what do you think, @MehmetC-1QBit ?

Reply:
@KatieO4t1qbit: 
1. With the idea to remove "the" from the headings, without “Solver” at the end of "Langevin Dynamics", it looks like we’re talking about Langevin dynamics _in general_ whereas the items below it are about machines/solvers. I think this is the case in general (that is, that "Solver" should be used), and have brought it up to Farhad.
2. Now, "Stochastic Differential Equations" is at the same lower _level_ of heading as "Example script...", whereas it was previously part of a title that was one level above in terms of headings. Should it not be one level of headings higher, even if then things are not "cleaner"?
3. "Stochastic Differential Equations", if it is acceptable to have it at a lower level of heading, should not be in title case, as the heading below it is not in title case (or the other way around).

Reply:
Thanks @KatieO4t1qbit. It looks nice.

Reply:
Thank you, both!
And thanks @MarkoB4t1qbit for the explanation. I've fixed these things in the latest commit!